tests/test_model_config_measurement.py

```diff
@@ -0,0 +1,145 @@
+# Copyright (c) 2022 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from tests.common.test_utils import convert_non_gpu_metrics_to_data
+from model_analyzer.record.metrics_manager import MetricsManager
+from model_analyzer.result.result_comparator import ResultComparator
+from model_analyzer.result.model_config_measurement import ModelConfigMeasurement
+
+import unittest
+from unittest.mock import MagicMock, patch
+from .common import test_result_collector as trc
+
+
+class TestModelConfigMeasurement(trc.TestResultCollector):
+
+    def setUp(self):
+        self.model_config_name = "modelA"
+        self.model_specific_pa_params = {"batch_size": 1, "concurrency": 1}
+
+        self.non_gpu_metric_values = {
+            "perf_throughput": 1000,
+            "perf_latency_p99": 20,
+            "cpu_used_ram": 1000
+        }
+
+        self.mcmA = self._construct_model_config_measurement(
+            self.model_config_name, self.model_specific_pa_params,
+            self.non_gpu_metric_values)
+
+        mcmB_non_gpu_metric_values = {
+            "perf_throughput": 2000,
+            "perf_latency_p99": 40,
+            "cpu_used_ram": 1000
+        }
+
+        self.mcmB = self._construct_model_config_measurement(
+            "modelB", self.model_specific_pa_params, mcmB_non_gpu_metric_values)
+
+    def tearDown(self):
+        NotImplemented
+
+    def test_init(self):
```

Comment:
What do these all test? Add test intent comment to every test.

Reply:
Done